tools/eslint-rules/rules-utils.js

```diff
@@ -13,6 +13,16 @@ module.exports.isRequired = function(node, modules) {
 };
 
 /**
+ * Return true if common module is present as a require call
+ * in AST
+ */
```

Comment:
This description is not entirely accurate. The given node must *be* a call to `require`, it does not work for nodes which contain such a call.

Reply:
@tniessen Please check now. Suggest if any corrections.